### PR TITLE
feat(PoS): add heimdall fee topup document

### PR DESCRIPTION
**Pull request by ashucoder9**

### CODE REVIEW

File: docs/pos/operate/validator/topup-heimdall-fee.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+:::
+
+In order to top up your Heimdall fee balance, we recommend Validators to follow the below provided steps:
+
+1. Head over to [Etherscan.io](https://etherscan.io)
+
+2. Enter the `StakeManagerProxy` contract address in the search box: [0x5e3Ef299fDDf15eAa0432E6e66473ace8c13D908](https://etherscan.io/address/0x5e3Ef299fDDf15eAa0432E6e66473ace8c13D908)
+
+3. Under the **Code** menu, select the **Write as Proxy** tab. COnnect your Web3 wallet using the **Connect to Web3** button.
```
Comment (ashucoder9) [inline docs/pos/operate/validator/topup-heimdall-fee.md:31]:
```suggestion
3. Under the **Code** menu, select the **Write as Proxy** tab. Connect your Web3 wallet using the **Connect to Web3** button.
```

File: docs/pos/operate/validator/topup-heimdall-fee.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+This document provides instructions on how to topup the Heimdall fee for a PoS Validator. This guide requires basic experience with Etherscan along with key details like Validator signer address.
+
+:::info
+
+We are working to add the **Topup Heimdall Fee** feature in the Staking UI. The instructions will be available here once the feature is live.
+
+:::
+
+In order to top up your Heimdall fee balance, we recommend Validators to follow the below provided steps:
```
Comment (ashucoder9) [inline docs/pos/operate/validator/topup-heimdall-fee.md:25]:
```suggestion
In order to top up your Heimdall fee balance, we recommend Validators follow the below steps:
```

File: docs/pos/operate/validator/topup-heimdall-fee.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  - polygon wiki
+  - polygon
+  - validator
+  - topup
+  - heimdall
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+This document provides instructions on how to topup the Heimdall fee for a PoS Validator. This guide requires basic experience with Etherscan along with key details like Validator signer address.
```
Comment (ashucoder9) [inline docs/pos/operate/validator/topup-heimdall-fee.md:17]:
```suggestion
This document provides instructions on how to top up the Heimdall fee for a PoS Validator. This guide requires basic experience with Etherscan along with key details like the Validator signer address.
```

File: docs/pos/operate/validator/topup-heimdall-fee.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+1. Head over to [Etherscan.io](https://etherscan.io)
+
+2. Enter the `StakeManagerProxy` contract address in the search box: [0x5e3Ef299fDDf15eAa0432E6e66473ace8c13D908](https://etherscan.io/address/0x5e3Ef299fDDf15eAa0432E6e66473ace8c13D908)
+
+3. Under the **Code** menu, select the **Write as Proxy** tab. COnnect your Web3 wallet using the **Connect to Web3** button.
+
+    <img src={useBaseUrl("img/connect-wallet-etherscan.png")} />
+
+4. Scroll down to `topUpForFee` method (#26 in the list) and select it. You will see something like the below screenshot.
```
Comment (ashucoder9) [inline docs/pos/operate/validator/topup-heimdall-fee.md:35]:
```suggestion
4. Scroll down to the `topUpForFee` method (#26 in the list) and select it. You will see something like the below screenshot.
```

**Review by 0xsajal — APPROVED** [review]

File: docs/pos/operate/validator/topup-heimdall-fee.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+    <img src={useBaseUrl("img/connect-wallet-etherscan.png")} />
+
+4. Scroll down to the `topUpForFee` method (#26 in the list) and select it. You will see something like the below screenshot.
+
+    <img src={useBaseUrl("img/topup-heimdall-fee.png")} />
+
+5. Fill in the details:
+
+    - `user`: Validator's Signer Address
+    - `heimdallFee`: Topup fee (**minimum 1 ETH**)
```
Comment (ashucoder9) [inline docs/pos/operate/validator/topup-heimdall-fee.md:42]:
```suggestion
    - `heimdallFee`: Topup fee (**minimum 1 MATIC**)
```

**Comment by sunil0410** [comment]
Good to go

**Review by mt-polygon-technology — APPROVED** [review]

**Review by mt-polygon-technology — APPROVED** [review]

**Review by mt-polygon-technology — APPROVED** [review]

**Pull request merged**